packages/synapse-react-client/src/mocks/team/mockTeam.ts

```diff
@@ -0,0 +1,160 @@
+import { Team, TeamMembershipStatus } from '@sage-bionetworks/synapse-types'
+import {
+  MOCK_USER_ID,
+  MOCK_USER_ID_2,
+  MockUserOrTeamData,
+} from '../user/mock_user_profile'
+import { ACT_TEAM_ID } from '../../utils/SynapseConstants'
+
+export const MOCK_TEAM_ID = 987654
+export const MOCK_TEAM_ID_2 = 987655
+export const MOCK_TEAM_ID_3 = 987656
+export const MOCK_TEAM_ID_4 = 987657
+export const MOCK_CHALLENGE_PARTICIPANT_TEAM_ID = 987658
+
+export const mockTeamData: Readonly<Team> = {
+  id: String(MOCK_TEAM_ID),
+  name: 'Mock Team',
+  description: 'A team that already has super cool fake users',
+  icon: '',
+  canPublicJoin: true,
+  canRequestMembership: true,
+  etag: 'f29b79d6-5b63-4641-93c7-30d954b4328c',
+  createdOn: '2013-11-02T01:01:18.373Z',
+  modifiedOn: '2019-01-31T20:34:40.057Z',
+  createdBy: String(MOCK_USER_ID),
+  modifiedBy: String(MOCK_USER_ID_2),
+}
+
+export const mockTeamData2: Readonly<Team> = {
+  id: String(MOCK_TEAM_ID_2),
+  name: 'Mock team public can join',
+  description: 'A team for fake users to join',
+  icon: '',
+  canPublicJoin: true,
+  canRequestMembership: false,
```

Comment:
Note that each team has a different value for `canPublicJoin` and `canRequestMembership`. This will be useful in a future change where we can test each scenario when trying to join a challenge submission team